ibmmq/json_schemas/channel.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/ibmmq/bindings/channel.json",
+  "title": "Channel Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^x-[\\w\\d\\.\\-\\_]+$": {
+      "$ref": "https://github.com/asyncapi/asyncapi-node/blob/master/schemas/2.0.0.json#/definitions/specificationExtension"
+    }
+  },
+  "properties": {
+    "destinationType": {
+      "type": "string",
+      "enum": ["topic", "queue"]
+    },
+    "queue": {
```

Comment:
can be present only if `destinationType=queue` so you need to use `oneOf`/`allOf` again